Update README.md (#2180)
huggingface/trl: README.md

> # TRL - Transformer Reinforcement Learning

Reviewer: any motivation to move the banner down?

Author: Like added this as a heading

## What is it?

> TRL is a library that post-trains LLMs and diffusion models using methods such as Supervised fine tuning (SFT), Proximal Policy Optimization (PPO), and Direct Preference Optimization (DPO). 

Reviewer: I'd revert this one

Reviewer: ```suggestion
TRL is a library that post-trains LLMs and diffusion models using methods such as Supervised Fine-Tuning (SFT), Proximal Policy Optimization (PPO), and Direct Preference Optimization (DPO).
```

- **`Efficient and scalable`**: 
    - [🤗 Accelerate](https://github.com/huggingface/accelerate) is the backbone of TRL that models training to scale from a single GPU to a large-scale multi-node cluster with methods such as DDP and DeepSpeed.

>     - [`PEFT`](https://github.com/huggingface/peft) is fully integrated and allows to training of even the largest models on modest hardware with quantization and methods such as LoRA or QLoRA.

Reviewer 2: ```suggestion
    - [`PEFT`](https://github.com/huggingface/peft) is fully integrated and allows users to train even the largest models on modest hardware with quantization and methods such as LoRA or QLoRA.
```

### From source

> If you want to use the latest features before an official release you can install them from source:

Reviewer: Here, you install the lib, not the features

> If you want to use the latest features before an official release, you can install TRL from the source:

Reviewer: ```suggestion
If you want to use the latest features before an official release, you can install TRL from source:
```

Reviewer 2: +1
